x/genutil/client/cli/gentx.go

```diff
@@ -26,13 +30,27 @@ import (
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	"github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
```

Comment:
ditto https://github.com/cosmos/cosmos-sdk/pull/7279#discussion_r501109927

Reply:
Thanks for the review and apologizes for the delay in responding :bow: 

This change just maintains backwards compatibility, based on this discussion => https://github.com/cosmos/cosmos-sdk/pull/7279#discussion_r487128039 && this PR https://github.com/cosmos/cosmos-sdk/pull/7316